src/SFA.DAS.ApplyService.Web/Controllers/Roatp/ApplicationController.cs

```diff
@@ -44,7 +44,7 @@ public async Task<IActionResult> DeleteFile(Guid Id, int sequenceNo, int section
 
             await _qnaApiClient.DeleteFile(applicationId, selectedSection.Id, pageId, questionId, filename);
 
-            return RedirectToAction("Page", "RoatpApplication", new { applicationId, sequenceId = sequenceNo, sectionId, pageId, redirectAction });
+            return RedirectToAction("Page", "RoatpApplication", new { applicationId, sequenceId = sequenceNo, sectionId, pageId, redirectAction = __redirectAction });
```

Comment:
This rename isn't needed. You can just pass redirectAction from the controller action